website/docs/r/virtual_machine.html.markdown

```diff
@@ -1391,15 +1391,15 @@ The configuration looks similar to the one below:
 
 ```hcl
 resource "vsphere_virtual_machine" "vm" {
-  ...
+  # ... other configuration ...
 
-  clone {
+  clone = {
     template_uuid = "${data.vsphere_virtual_machine.template_from_ovf.id}"
   }
 
-  vapp {
-    properties {
-      "guestinfo.tf.internal.id" = "42"
+  vapp = {
```

Comment:
I looked for code examples of this on github, since I don't have a way to test locally, and I'm seeing it used without the equal sign.

```suggestion
  vapp {
```